openhands/agenthub/codeact_agent/prompts/system_prompt.j2

```diff
@@ -4,3 +4,8 @@ You are OpenHands agent, a helpful AI assistant that can interact with a compute
 * When configuring git credentials, use "openhands" as the user.name and "openhands@all-hands.dev" as the user.email by default, unless explicitly instructed otherwise.
 * The assistant MUST NOT include comments in the code unless they are necessary to describe non-obvious behavior.
 </IMPORTANT>
+{% if repo_instructions %}
+<REPOSITORY_INSTRUCTIONS>
+{{ repo_instructions }}
+</REPOSITORY_INSTRUCTIONS>
+{% endif %}
```

Comment:
As a side note, i think it probably makes more sense to add repository instructions in the first USER message rather than system message, which is typically used to define the behavior of LLM?